lib/ansible/config/base.yml

```
@@ -863,8 +863,8 @@ DEFAULT_MODULE_COMPRESSION:
   env: []
   ini:
   - {key: module_compression, section: defaults}
-# vars:
-#   - name: ansible_module_compression
+ vars:
+   - name: ansible_module_compression
```

```suggestion
  vars:
    - name: ansible_module_compression
```
It should be indented with one more space